docs/source/development/design/defining_new_models.md

```diff
@@ -526,9 +577,24 @@ def _setup(self, pond_data_path: Path, constants: FreshwaterConstants) -> None:
     )
 ```
 
+:::{admonition} Additional data inputs to a model
+:class: tip
+
+Most of the data in a Virtual Ecosystem simulation is loaded into the central `Data`
+object and shared between the models. However, you may need to load additional data to
+initialise your model that is only used within the model and not shared through the
+`Data` object.
+
+It is absolutely fine to configure and load data like this within the model `_setup`
```

Comment:
I would take the actual data loading outside of `_setup` and do it in `from_config` and pass loaded objects (numpy arrays, xarrays or whatever). As it is now, it is impossible to create an instance of those models without having an actual file with valid data lying around. It is generally considered an [anti-pattern](https://dzone.com/articles/fixing-the-constructor-anti-pattern).